exercises/practice/leap/.docs/instructions.md

```diff
@@ -0,0 +1,24 @@
+# Description
```

Comment:
When I run `./bin/configlet sync --update --exercise leap` the headline is `Instructions` instead of `Description` is the change intended?

Reply:
I would have copied that from somewhere else ... fsharp I think. It may not matter but it's something to attempt consistency on. I'd prefer to do what configlet suggests.

Reply:
> I'd prefer to do what configlet suggests.

Yes, always go with what `configlet` does!